app/pages/projects.cjsx

```diff
@@ -22,14 +22,12 @@ module.exports = React.createClass
     query = {include: 'avatar'}
 
     if !apiClient.params.admin
-      query.launch_approved = true
+      query.homepage = true
```

Comment:
What's `homepage` mean here? This is the projects page. Still looks like the expected response though.